### ISSUE TITLE
Proposed changes to Boundary distinctness

### ISSUE THREAD

**Issue by lauren-obrien** [manaakiwhenua/NZ_soildesc_2024#2]
An upper limit is proposed on Diffuse boundaries of 30cm, after which it is suggested a new horizon should be defined. This has not been explicitly asked for in other standards, which keep the diffuse boundary open-ended. 

Seeking thoughts on whether this is a positive change, and/or whether the 30cm cutoff is optimal. We don't want excessive horizon numbers.

**Comment by SoilNotDirt** [comment]
I would prefer getting rid of diffuse, meaning boundaries exceeding 10 cm become a transition horizon.

**Comment by lynni1950** [comment]
I agree, anything over 10 cm is a horizon in it's own right, i.e., a transition horizon
